node/pkg.sls

```diff
@@ -8,7 +8,7 @@ nodejs.ppa:
       - pkgrepo: nodejs.ppa
   pkgrepo.managed:
     - humanname: NodeSource Node.js Repository
-    - name: deb {{ salt['pillar.get']('node:ppa:repository_url', 'https://deb.nodesource.com/node_0.12') }} {{ grains['oscodename'] }} main
+    - name: deb {{ salt['pillar.get']('node:ppa:repository_url', 'https://deb.nodesource.com/node_6.x') }} {{ grains['oscodename'] }} main
```

Comment:
you should change this to be in map.jinja instead of hardcoding it to prevent having to do this exact PR in the future